src/App.tsx

```diff
@@ -20,36 +22,44 @@ import {
 import DropdownBody from "./components/DropdownBody/DropdownBody";
 import x from "./assets/fonts/Roboto/Roboto-Regular.ttf";
 import { useState } from "react";
+import FilterBar from "./components/FilterBar/FilterBar";
 function App() {
   const [visibleBigTitle, setVisibleBigTitle] = useState(true);
+  const [appHeaderFilter, setAppHeaderFilter] =
+    useState<string>("Top Headlines");
+
   return (
-    <AppContainer>
-      <AppHeader />
-      <MainContainer>
-        <FilterContainer>
-          <FilterDropDown placeholder="Country" selectOptions={selectOptions} />{" "}
-          <FilterDropDown placeholder="Country" selectOptions={selectOptions} />{" "}
-          <FilterDropDown placeholder="Country" selectOptions={selectOptions} />
-          <FilterDropDown placeholder="Country" selectOptions={selectOptions} />
-        </FilterContainer>
-        {visibleBigTitle ? (
-          <TitleHeadLines>Top Headlines in Israel</TitleHeadLines>
-        ) : (
-          <p> 30 seraches</p>
-        )}
-        <MainContent>
-          <ArticleContainer>
-            {articles.map((article, index) => (
-              <ArticleCard article={article} key={index} />
-            ))}
-          </ArticleContainer>
-          <DashboardContainer>
-            <TempDashboard>test12344567</TempDashboard>
-            <TempDashboard>test</TempDashboard>
-          </DashboardContainer>
-        </MainContent>
-      </MainContainer>
-    </AppContainer>
+    <>
+      <AppContext.Provider
+        value={{
+          appHeaderFilter,
+          setAppHeaderFilter,
+        }}
+      >
+        <AppContainer>
+          <AppHeader />
+          <MainContainer>
+            <FilterBar />
+            {visibleBigTitle ? (
+              <TitleHeadLines>Top Headlines in Israel</TitleHeadLines>
+            ) : (
+              <p> 30 seraches</p>
+            )}
```

Comment:
can that data be saved in the same state, and just change it depending on the everything | top headlines?
```suggestion
           <p>{articlesTitle}</p> // THAT WILL BE A STATE
```